submissions/Roophee/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,428 @@
+                    </tr>
+                </tbody></table>
+            </article>
+        </section>
+        <section class="content__video">
+            <div class="content__video-label">
+                <P>
+                    Смотреть трейлер
+                </P>
+            </div>
+            <div class="content__video-trailer hide">
+            <iframe width="560" height="315" src="https://www.youtube.com/embed/R0Ex4-kZWaI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+        </section>
+        <main class="content__seats">
+            <div class="content__seats-container">
+                <div class="seats__label">
+                    <h2>
+                        Заказать билеты
+                    </h2>
+                </div>
+                <form class="seats__body">
+                    <table class="seats__body-table">
+                        <thead>
+                            <tr>
+                                <td  class="seats__body-table-head-row">
+                                </td>
+                                <td class="seats__body-table-head-seat">
+                                    Экран
+                                </td>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <th class="seats__body-table-row">&#160</th>
+                                <td class="seats__body-table-seats">
+                                    <div class="screen"></div>
+                                </td>
+                                </tr>
+                                <tr>
+                                    <th class="seats__body-table-row">Ряд</th>
+                                    <td class="seats__body-table-seats weight400">Место
+                                    </td>
+                                    </tr>
+                            <tr>
+                            <th class="seats__body-table-row">1</th>
+                            <td class="seats__body-table-seats">
+                                <input class="seats__body-table-seats-place" type="checkbox" id="r1_s1" value="r1_s1" name="ticketSeat"><label class="whitespaces" for="r1_s1">1</label>
```

Comment:
Add `cursor: pointer` to all interactive elemnts.

Reply:
Done.